style/amenity-points.mss

```diff
@@ -1371,11 +1371,19 @@
     marker-clip: false;
   }
 
-  [feature = 'natural_saddle'][zoom >= 15] {
+  [feature = 'natural_saddle'][zoom >= 15],
+  [feature = 'natural_saddle'][mountain_pass = 'yes'][zoom >= 14] {
     marker-file: url('symbols/natural/saddle.svg');
     marker-fill: @landform-color;
     marker-clip: false;
   }
+  
+  [feature = 'mountain_pass'][zoom >= 15] {
+    marker-file: url('symbols/mountain_pass.svg');
+    marker-fill: none;
+    marker-line-color: @transportation-icon;
```

Comment:
I would prefer this to be in landform-color or a neutral tone. It's not a transportation feature like a ferry, bus stop, bike share station, taxi stand, or parking lot, but rather a landcape feature: the highest elevation point where a highway crosses a ridgeline, so `@landform-color` might make sense semantically. Gray or black could also work if we consider this more like a man-made feature since the presence of a road or path is required.